docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -124,4 +123,117 @@ An {oldetection} configuration object has the following properties:
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+
+[discrete]
+[[regression-resources]]
+==== {regression-cap} configuration objects
+
+A `{regression}` configuration object has the following properties:
+
+`dependent_variable`::
+  (Required, string) Defines which field of the {dataframe} is to be predicted. 
+  This parameter is supplied by field name and must match one of the fields in 
+  the index being used to train. If this field is missing from a document that 
+  row will not be used for training, but a prediction with the trained model 
+  will be generated for it. The data type of the field must be numeric. It is 
+  also known as continuous target variable.  
+
+`eta`::
+  (Optional, double) The shrinkage applied to the weights. Smaller values result 
+  in larger forests which have better generalization error. However, the smaller 
+  the value the longer the training will take and the inference will also be 
+  larger and slower. For more information, see 
+  https://en.wikipedia.org/wiki/Gradient_boosting#Shrinkage[this wiki article] 
+  about shrinkage.
+
+`feature_bag_fraction`::
+  (Optional, double) Defines the fraction of features which will be used when 
+  selecting a random bag for each candidate split.
+  
+`maximum_number_trees`::
+  (Optional, integer) Defines the maximum number of trees the forest is 
+  allowed to contain. The maximum value is 2000.
+    
+`prediction_field_name`::
+  (Optional, string) Defines the name of the prediction field in the results. 
+  Defaults to `<dependent_variable>_prediction`.
+    
+`training_percent`::
+  (Optional, integer) Defines what percentage of the data rows that are 
+  eligible for training. Defaults to `100`.
+
+
+===== {regression-cap} regularization parameters
+
+Regularization parameters are constant multipliers of a regularization term 
+added to the optimization objective. They aim to prevent overfitting on the 
+training data set and improve generalization performance. These parameters are 
+calculated and set automatically by 
+<<ml-hyperparameter-optimization,hyperparameter optimization>> if you don't 
+supply their values manually. For more 
+information about regularization parameters and optimization, read the 
+https://arxiv.org/pdf/1603.02754.pdf[**XGBoost: A Scalable Tree Boosting System**] 
+paper. 
+
+`gamma`::
+  (Optional, double) It multiplies a linear penalty associated with the size of 
+  individual trees in the forest. The higher the value the more training will 
+  prefer smaller trees. This will reduce the number of distinct values the 
+  prediction function can take. To have an effect, `gamma` value must be 
+  comparable to the reduction in the expected loss for splitting the data. The 
+  smaller this parameter the larger individual trees will be. This will result 
+  in slower train time and larger models for which inference is slower.
```

Comment:
Since we say larger values result in smaller trees, how about:
"The smaller this parameter the longer train will take. It will also produce larger models for which inference is slower."